docs/discover/view-single-document.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[discover-view-single-document]]
+=== View single document
```

Comment:
This doc is so short.  Can we combine it with "View surrounding document"?

Reply:
Yes, it's short, question is, we would need a new title and intro for "View surrounding document" if we would combine it, right?

Reply:
Do any of these work?

- Manage the display of documents
- Customize the document display
- View documents in context

Reply:
How about 

### Additional ways to view a document

Some intro 

### View surrounding documents

### View single document

Do you think this would work?

Reply:
View a document

intro

- View single document
- View surrounding documents

Reply:
done!